### ISSUE TITLE
Two feature requests

### ISSUE THREAD

**Issue by BRNKR** [tonyrobotics/RoboWare#12]
Hey,

1. if you change the package name with "rename" function. is it possible to rename all of it in "package.xml" and CMakeList.txt?
2. Is it possible to add automatically the first line in python scripts when you create it? "#!/usr/bin/env python"

Thx in advance

**Comment by tony-robotics** [comment]
Yes, these two features will be added in upcoming releases, it will be released by the end of this month. Thanks for your advice!
